openshift/release/nightly-ci-run.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelines-triggers-nightly
+spec:
+  pipelineSpec:
+    workspaces:
+      - name: source
+    tasks:
+      - name: fetch-repository
+        taskRef:
+          name: git-clone
+        workspaces:
+          - name: output
+            workspace: source
+        params:
+          - name: url
+            value: https://github.com/openshift/tektoncd-triggers
+          - name: revision
+            value: master
+          - name: subdirectory
+            value: ""
+          - name: deleteExisting
+            value: "true"
+      - name: create-pr
+        runAfter:
+          - fetch-repository
+        workspaces:
+          - workspace: source
+            name: source
+        taskSpec:
+          workspaces:
+            - name: source
+          steps:
+            - name: create-pr
+              workingDir: $(workspaces.source.path)
+              env:
+                - name: HUB_VERSION
+                  value: "true"
+                - name: GITHUB_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: nightly-ci-github-hub-token
+                      key: hub-token
```

Comment:
From where do we get this secret :thinking:

Reply:
this is there on the qe cluster